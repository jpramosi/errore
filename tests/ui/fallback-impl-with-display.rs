use core::fmt::{self, Display};

use errore::*;

#[derive(Error, Debug)]
#[error]
pub struct MyError;

impl Display for MyError {
    fn fmt(&self, _formatter: &mut fmt::Formatter) -> fmt::Result {
        unimplemented!()
    }
}

fn main() {}
