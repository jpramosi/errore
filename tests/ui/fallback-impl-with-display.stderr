error: expected attribute arguments in parentheses: #[error(...)]
 --> tests/ui/fallback-impl-with-display.rs:6:3
  |
6 | #[error]
  |   ^^^^^

error[E0119]: conflicting implementations of trait `std::fmt::Display` for type `MyError`
 --> tests/ui/fallback-impl-with-display.rs:5:10
  |
5 | #[derive(Error, Debug)]
  |          ^^^^^ conflicting implementation for `MyError`
...
9 | impl Display for MyError {
  | ------------------------ first implementation here
  |
  = note: this error originates in the derive macro `Error` (in Nightly builds, run with -Z macro-backtrace for more info)
